src/App.js

```diff
@@ -1,37 +0,0 @@
-// import './App.css';
-// import { Switch } from "react-router";
-
-import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
-import Login from './Auth/Login/Login'
-import Register from "./Auth/Register/Register";
-import Reset from "./Auth/ResetPassword/ResetPassword";
-import Dashboard from "./Dashboard/Dashboard";
-import Header from "./UI/Header";
-import AddBlog from "./Blog/AddBlog";
-import Blogs from "./Blog/Blogs";
-import MyBlogs from "./Blog/MyBlogs";
-import Editblog from "./Blog/EditBlog";
-
-function App() {
-
-  return (
-
-
-    <Router>
```

Comment:
Do not directly use App.js as routes files, always use App.js only for import global store and routes.js file and theme related changes